jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ui/UiUtils.kt

```diff
@@ -197,3 +204,53 @@ class WrappingCellRenderer(private val wrapOnSelection: Boolean, private val tog
         return component
     }
 }
+
+// Generalized the renderer
+class ResizingColumnRenderer(showSeconds: Boolean? = null) : TableCellRenderer {
```

Comment:
When would show seconds be nullable?

Reply:
It would be nullable for columns that do not handle date/time. If you look at lines 231-238, this will handle the component text differently based on `formatter`.

Reply:
We can make this `abstract` then have a getText or something that is implemented by the child class, then we would not have to worry about this unrelated field